components/brave_vpn/browser/connection/win/brave_vpn_helper/brave_vpn_helper_crash_reporter_client.cc

```diff
@@ -124,18 +123,19 @@ int BraveVPNHelperCrashReporterClient::GetResultCodeRespawnFailed() {
 
 bool BraveVPNHelperCrashReporterClient::GetCrashDumpLocation(
     std::wstring* crash_dir) {
-  *crash_dir = install_static::GetCrashDumpLocation();
-  return !crash_dir->empty();
+  auto profile_dir = brave_vpn::GetVpnServiceProfileDir();
+  *crash_dir = (profile_dir.Append(L"Crashpad")).value();
+  return !profile_dir.empty();
 }
 
 bool BraveVPNHelperCrashReporterClient::GetCrashMetricsLocation(
     std::wstring* metrics_dir) {
-  install_static::GetUserDataDirectory(metrics_dir, nullptr);
+  *metrics_dir = brave_vpn::GetVpnServiceProfileDir().value();
   return !metrics_dir->empty();
 }
 
 bool BraveVPNHelperCrashReporterClient::IsRunningUnattended() {
-  return true;
+  return false;
```

Comment:
why is this false?

Reply:
https://source.chromium.org/chromium/chromium/src/+/main:components/crash/core/app/crash_reporter_client.cc;bpv=1;bpt=1